lib/device/device.dart

```diff
@@ -0,0 +1,231 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:html' as html;
+
+import 'package:vm_service_lib/vm_service_lib.dart';
+
+import '../framework/framework.dart';
+import '../globals.dart';
+import '../ui/elements.dart';
+
+// TODO: set toggle values on a full restart (when we see a new isolate)
```

Comment:
nit: replace all TODOs with
TODO(devonc):
to disambiguate from future TODO(jacobr)